source/diagnostics.tex

```diff
@@ -939,6 +849,8 @@
 such type. \end{note}
 
 \indexlibrary{\idxcode{error_category}}%
+\indexlibrary{\idxcode{error_category}!constructor}%
+\indexlibrary{\idxcode{error_category}!destructor}%
```

Comment:
Moving the indexes to here seems a bit odd, but so does having decls in the synopsis with no corresponding itemdecls.